Assembler.Base/DefaultAssemblerFactory.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
+using Assembler.Base.Creators;
+using Assembler.Base.FrameHandlers;
+using Assembler.Base.MessageEnrichers;
+using Assembler.Base.Releasing;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+using Assembler.Core.RawAssemblingEntities;
+using Microsoft.Extensions.Logging;
+
+namespace Assembler.Base
+{
+    public class DefaultAssemblerFactory<TFrame, TMessageInAssembly>
```

Comment:
I will go over this after everything will be finished, dont want to do double work